examples/seismic/tti/tti_example3D.py

```diff
@@ -57,18 +57,19 @@ def setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     rec = Receiver(name='rec', ntime=nt, coordinates=receiver_coords)
 
     return AnisotropicWaveSolver(model, source=src, time_order=time_order,
-                                 space_order=space_order, receiver=rec, dse=dse)
+                                 space_order=space_order, receiver=rec, dse=dse, dle=dle,
+                                 compiler=compiler)
```

Comment:
This was being passed to `Operator.apply()` from [here](https://github.com/opesci/devito/blob/master/examples/benchmark.py#L32) and eaten up since the previous iteration of `arguments` didn't croak on unknown `kwargs` being passed, it simply ignored them and moved on. When I added the `raise Exception()`, I saw incorrect arguments being passed in a lot of tests/examples. I simply moved this argument from the `apply` call to the constructor call.